icp/EXAMPLES.md

```
@@ -2,6 +2,196 @@
+    apiVersion: traefik.containo.us/v1alpha1
+    kind: Middleware
+    metadata:
+      name: icp-sticky
+    spec:
+      affinity:
+        sticky:
+          cookie:
+            name: "ICP_AFFINITY"
+            httpOnly: true
+    ```
+   - Consult your Gateway controller's documentation for cookie-based sticky sessions.
```

markdownlint-cli2 (0.20.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 3

(MD005, list-indent)